doc/isCurrent.md

```diff
@@ -0,0 +1,67 @@
+@function can-route.isCurrent isCurrent
+@parent can-route.static
+
+@description Check if data represents the current route.
+
+@signature `route.isCurrent(data [,subsetMatch] )`
+
+  Compares `data` to the current route. Used to verify if an object is
+  representative of the current route.
```

Comment:
Can you describe what the following example does?  It's a bit too long for someone to instantly know.